reports-scheduler/build.gradle

```diff
@@ -40,7 +40,7 @@ buildscript {
         classpath "${opensearch_group}.gradle:build-tools:${opensearch_version}"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlin_version}"
         classpath "org.jetbrains.kotlin:kotlin-allopen:${kotlin_version}"
-        classpath "io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.12.0"
+        // classpath "io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.12.0"
```

Comment:
It appears the CVE is fixed in Detekt 1.22.x
https://detekt.dev/changelog#dependency-updates
`Update dependency org.yaml:snakeyaml to v1.33 - [#5354](https://github.com/detekt/detekt/pull/5354)`

The CVE's listed in description indicate the SnakeYaml DOS vuln is up-to(excluding) 1.32.


The Gradle plugin appears to be lockstep w/ Detekt versions
https://detekt.dev/docs/gettingstarted/gradle/#kotlin-dsl-1
`classpath("io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.22.0-RC2")`

Could you try updating the plugin version and see if it pulls the proper version of SnakeYaml ?

Reply:
Thanks for checking @pjfitzgibbons , I am running into build failures so will disable Detekt for the time being and upgrade the Detekt version with #520